boards/arm/sam34/arduino-due/src/sam_bringup.c

```diff
@@ -117,5 +125,19 @@ int sam_bringup(void)
 #endif
 
   UNUSED(ret);
```

Comment:
Let's move `UNUSED(ret);` after `#ifdef HAVE_LEDS` block

Reply:
Thanks for catching- fixed.